tests/csm/rest/test_csm_load.py

```diff
@@ -236,97 +216,6 @@ def test_44788(self):
         self.log.info("##### Test completed -  %s #####", test_case_name)
 
 
-    # pylint: disable=too-many-statements
```

Comment:
so we won't require this test at all even in future?

Reply:
moved it to tests/csm/rest/test_account_capacity.py